src/api/status/public/js/build-log/api.js

```diff
@@ -3,19 +3,23 @@ const autodeploymentUrl = (path) => `//${location.hostname}:4000/${path}`;
 
 export const checkBuildStatus = async () => {
   try {
-    const res = await fetch(autodeploymentUrl('status'));
+    const res = await fetch(autodeploymentUrl('status'), {
+      headers: {
+        'Cache-Control': 'no-cache',
```

Comment:
This needs to happen on the response vs. request.  Do this in the autodeployment server here: https://github.com/Seneca-CDOT/telescope/blob/master/tools/autodeployment/builds.js#L100-L129

You can add this to each of those before you send the result:

```js
res.setHeader('Cache-Control', 'no-cache');
```

Reply:
NOTE: this can (and probably should) happen in a separate PR so we don't block this work.  I would just remove the bit above to add headers here for now, and do a second PR.

Reply:
Ok, I'll make another PR today.